### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#63]
### What work do we need to do?
The design system service needs to be inclusive and confirm to WCAG 2.1.

### Why are we doing it?
It makes things better. It's the right thing to do.

### Related stories

- [x] Fix: The service name in the header doesn't have a hover [#64 ](https://github.com/dwp/design-system/issues/64)
- [x] Fix: Missing skip to main content link [#65 ](https://github.com/dwp/design-system/issues/65)
- [x] Fix: Visited link colour contrast [#66](https://github.com/dwp/design-system/issues/66)
- [x] Feature: Mobile navigation
